lib/ui/painting.dart

```diff
@@ -1568,6 +1568,26 @@ class Paint {
   }
 }
 
+/// The color space describes the colors that are available to an [Image].
+///
+/// This value can help decide which [ImageByteFormat] to use with
+/// [Image.toByteData], where images that are in the [extendedSRGB] color space
+/// should use something like [ImageByteFormat.rawExtendedRgba128] so that
```

Comment:
```suggestion
/// should use something like [ImageByteFormat.rawExtendedRgba128] so that
```

Nit, use stronger phrasing. Consider something like:

"An image in the extendedSRGB color space must use ImageByteFormat.rawExtendedRgba128 to avoid losing colors. Using ImageByteFormat blah blah will cause loss of color information" And define what loss of color information means if its a well defined process

Reply:
Okay, reworded and I think that information would be better to be situated at `toByteData` so I added it there.